pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -544,6 +553,60 @@ func getRPSUnitOptions(rpsMask string) []*unit.UnitOption {
 	}
 }
 
+func getTunedOneShotSystemdUnit() string {
```

Comment:
I'd prefer have this as loadable (embeddable?) asset rather than a string constant. WDYT? What do other reviewers think?

Reply:
The other units are created programatically even. I do not care much, but an embedded asset it indeed a nice idea.

Reply:
Creating the unit programatically doesn't make sense in this case in my view.  However, making this a loadable asset (if you mean from a new file within a repo) would make sense to me.  Any suggestions as to where to keep the asset?

Reply:
`assets/performanceprofile/configs` works. In the future we should probably further split the systemd unit templates in their own directory.